firebase-ml-modeldownloader/firebase-ml-modeldownloader.gradle

```diff
@@ -25,7 +25,7 @@ android {
     compileSdkVersion project.targetSdkVersion
 
     defaultConfig {
-        minSdkVersion project.minSdkVersion
+        minSdkVersion 16
```

Comment:
The existing sdk we are replacing was also min 16, so keeping this limit seemed appropriate. I believe the specific requirement in this PR is the addition of the android.util.LongSparseArray - which requires 16 instead of 14.

Reply:
Got it, thanks!